src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs

```diff
@@ -0,0 +1,905 @@
+        {
+            ParseOperationResult result = TryParseExactInternal(s, format, provider, style, out TimeOnly timeOnly);
+            if (result != ParseOperationResult.Success)
+            {
+                switch (result)
+                {
+                    case ParseOperationResult.WrongStyles: throw new ArgumentException(SR.Argument_InvalidDateStyles, nameof(style));
+                    case ParseOperationResult.ParseFailure: throw new FormatException(SR.Format(SR.Format_BadTimeOnly, s.ToString()));
+                    default:
+                        Debug.Assert(result == ParseOperationResult.WrongParts);
+                        throw new FormatException(SR.Format(SR.Format_DateTimeOnlyContainsNoneDateParts, s.ToString(), nameof(TimeOnly)));
+                }
+            }
+
+            return timeOnly;
+        }
+
+        /// <summary>
+        /// Converts the specified span representation of a time to its TimeOnly equivalent using the specified array of formats.
+        /// The format of the string representation must match at least one of the specified formats exactly or an exception is thrown.
+        /// </summary>
+        /// <param name="s">A span containing the characters that represent a time to convert.</param>
+        /// <param name="formats">An array of allowable formats of s.</param>
+        /// <returns>An object that is equivalent to the time contained in s, as specified by format, provider, and style.</returns>
+        public static TimeOnly ParseExact(ReadOnlySpan<char> s, string[] formats) => ParseExact(s, formats, null, DateTimeStyles.None);
+
+        /// <summary>
+        /// Converts the specified span representation of a time to its TimeOnly equivalent using the specified array of formats, culture-specific format information, and style.
+        /// The format of the string representation must match at least one of the specified formats exactly or an exception is thrown.
+        /// </summary>
+        /// <param name="s">A span containing the characters that represent a time to convert.</param>
+        /// <param name="formats">An array of allowable formats of s.</param>
+        /// <param name="provider">An object that supplies culture-specific formatting information about s.</param>
+        /// <param name="style">A bitwise combination of enumeration values that indicates the permitted format of s. A typical value to specify is None.</param>
+        /// <returns>An object that is equivalent to the time contained in s, as specified by format, provider, and style.</returns>
+        public static TimeOnly ParseExact(ReadOnlySpan<char> s, string[] formats, IFormatProvider? provider, DateTimeStyles style = DateTimeStyles.None)
+        {
+            ParseOperationResult result = TryParseExactInternal(s, formats, provider, style, out TimeOnly timeOnly);
```

Comment:
Can this call to the `ParseExtact` override that takes a `Span<char>` for the formats?